streampark-flink/streampark-flink-connector/pom.xml

```diff
@@ -48,6 +48,7 @@
         <module>streampark-flink-connector-jdbc</module>
         <module>streampark-flink-connector-kafka</module>
         <module>streampark-flink-connector-mongo</module>
+        <module>streampark-flink-connector-redis</module>
```

Comment:
Hi @1996fanrui , thanks for your clarification.Since StreamPark-Flink-connector-Redis relies on Flink-connector-Redis, which only provides scala2.11, this module is excluded from the Module list.
![image](https://user-images.githubusercontent.com/45427129/194497802-0d5e19c8-c7c0-47bf-828b-c0303eb7fde6.png)

Reply:
Thanks for you reply. If so, I will close this pr and issue.